Calibration/HcalIsolatedTrackReco/src/EcalIsolatedParticleCandidateProducer.cc

```diff
@@ -211,8 +211,8 @@ EcalIsolatedParticleCandidateProducer::produce(edm::Event& iEvent, const edm::Ev
   // is put into the Event
   
 //  std::cout<<"put cand into event"<<std::endl;
-  std::auto_ptr<reco::IsolatedPixelTrackCandidateCollection> pOut(iptcCollection);
-  iEvent.put(pOut);
+  std::unique_ptr<reco::IsolatedPixelTrackCandidateCollection> pOut(iptcCollection);
+  iEvent.put(std::move(pOut));
 
```

Comment:
@ghellwig I didn't use make_unique because I was doing several thousand edits across all of CMSSW and it was just too much work. You are correct, of course, that make_unique would be better.